src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.1.0
+-----
+
+ * Added `Controller:json` to simplify creating JSON responses when using the serializer component
```

Comment:
There is such a mix of capitalisation and full stops in this fill, I hope this is the expected format.